doc/man/8/rbd.rst

```diff
@@ -747,6 +747,9 @@ Per mapping (block device) `rbd device map` options:
   requests will fail with -EOPNOTSUPP, write zeroes requests will fall back to
   manually zeroing.
 
+* abort_on_full - Return -ENOSPC when the pool is full, or reaches quota, rather
+  than suspend IO (since 4.21, default is false).
+
```

Comment:
I'd go with something like
```
* abort_on_full - Fail write requests with -ENOSPC when the cluster is full or
  the data pool reaches its quota (since 4.21).  The default behaviour is to
  block until the full condition is cleared.
```